features/providers/step_definitions/civil_journey_steps.rb

```diff
@@ -128,6 +128,13 @@
   end
 end
 
+Given('I select Check Your Answers Change link for {string}') do |field_name|
+  field_name.downcase!
+  field_name.gsub!(/\s+/, '_')
+  # Used where there are duplicated id fields
+  find("#app-check-your-answers__#{field_name}:nth-child(1)").click_link('Change')
```

Comment:
Should there be elements with duplicated ids? I think ids should be unique on the page. It might be better to change the way the html is generated to prevent the duplication, rather than add a workaround here.